docs/source/ruletypes.rst

```diff
@@ -419,12 +419,6 @@ run_enhancements_first
 ``run_enhancements_first``: If set to true, enhancements will be run as soon as a match is found. This means that they can be changed
 or dropped before affecting realert or being added to an aggregation. (Optional, boolean, default false)
```

Comment:
I would add some more color here to indicate the nuance of the behavior we discussed.